torch/csrc/jit/codegen/cuda/lower_utils.h

```diff
@@ -75,6 +79,55 @@ const TensorView* asConstTV(const Val* const);
 
 bool isUnrolledFor(const Expr*);
 
+/*
```

Comment:
nit: Please use // comments (we already have a bit of mix, but it would be easier to stick to one style of comments for new code)

Reply:
Thanks. Changed.